tests/InitCommandTest.php

```diff
@@ -926,4 +930,50 @@ public function testRunWithClerkAdditionalAdminsWithoutDefaultAdmin(): void
             ->expectsConfirmation('Do you want to uninstall project-initializator package?')
             ->assertExitCode(0);
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testDefaultAdminsCredentials()
+    {
+        $this->mockNativeFunction(
+            'RonasIT\ProjectInitializator\Commands',
+            $this->mockAdminDefaultPassword('123456'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_default_admin.php', $this->getFixture('users_default_admin_default_credentials.php')),
+            $this->mockAdminDefaultPassword('654321'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_telescope_admin.php', $this->getFixture('telescope_users_default_credentials.php')),
+            $this->mockAdminDefaultPassword('123456'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_nova_admin.php', $this->getFixture('nova_users_default_credentials.php')),
+            $this->mockAdminDefaultPassword('123456'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_default_admin.php', $this->getFixture('admins_add_default_admin_default_credentials.php')),
+            $this->mockAdminDefaultPassword('654321'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_telescope_admin.php', $this->getFixture('admins_add_telescope_admin_default_credentials.php')),
+            $this->mockAdminDefaultPassword('123456'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_nova_admin.php', $this->getFixture('admins_add_nova_admin_default_credentials.php')),
+        );
+
+        $commandMock = $this
+            ->getMockBuilder(InitCommand::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $commandMock->method('ask')->willReturnCallback(fn ($question, $default) => $default);
+
+        $authTypeProperty = new ReflectionProperty(InitCommand::class, 'authType');
+        $authTypeProperty->setAccessible(true);
+
+        $createAdminMethod = new ReflectionMethod(InitCommand::class, 'createAdminUser');
+
+        $authTypeProperty->setValue($commandMock, AuthTypeEnum::None);
+
+        $createAdminMethod->invokeArgs($commandMock, ['my-app']);
```

Comment:
I belive it enough to test that the default answer is correct. No need to generate and assert migrations and test cases when admin name is not entering at all ar generated with the prefix (Telescope admin, Nova admin, etc.)

```suggestion
        $credentials = $createAdminMethod->invokeArgs($commandMock, ['my-app']);
        
        $this->assertEquals($credentials['name'], 'Admin');
```